swagger.yaml

```diff
@@ -139,16 +132,20 @@ parameters:
     required: true
     description: '"Bearer [token]" where [token] is your OAuth2 access token'
 definitions:
-  ResultObject:
+  ResultObjects:
     properties:
       data:
         type: array
         items:
           $ref: "#/definitions/ResourceObject"
+  ResultObject:
+    properties:
+      data:
+        $ref: "#/definitions/ResourceObject"
   ResourceObject:
```

Comment:
For `ResourceObject`, `ResultObject` and `ResultObjects`, I'd suggest rename them more specificly.

Reply:
Done